ethstats/ethstats_test.go

```diff
@@ -0,0 +1,63 @@
+package ethstats
+
+import (
+	"strconv"
+	"testing"
+)
+
+func TestParseEthstatsURL(t *testing.T) {
+	cases := []struct {
+		url              string
+		node, pass, host string
+	}{
+		{
+			url:  `"debug meowsbits":mypass@ws://mordor.dash.fault.dev:3000`,
+			node: "debug meowsbits", pass: "mypass", host: "ws://mordor.dash.fault.dev:3000",
+		},
+		{
+			url:  `"debug @meowsbits":mypass@ws://mordor.dash.fault.dev:3000`,
+			node: "debug @meowsbits", pass: "mypass", host: "ws://mordor.dash.fault.dev:3000",
+		},
+		{
+			url:  `"debug: @meowsbits":mypass@ws://mordor.dash.fault.dev:3000`,
+			node: "debug: @meowsbits", pass: "mypass", host: "ws://mordor.dash.fault.dev:3000",
+		},
+		{
+			url:  `name:@ws://mordor.dash.fault.dev:3000`,
+			node: "name", pass: "", host: "ws://mordor.dash.fault.dev:3000",
+		},
```

Comment:
`name@ws://mordor.dash.fault.dev` should also work (without the `:`)

Reply:
At least if it worked before...

Reply:
Roger. Not sure if it did before... but it does [now](https://github.com/ethereum/go-ethereum/pull/21640/commits/87e6d22040aaefe75d99443bffe405b10cf2f66f) :joy: